scenarios/kubernetes_e2e.py

```diff
@@ -313,28 +339,15 @@ def main(args):
     cluster = args.cluster or 'e2e-gce-%s-%s' % (
         os.environ['NODE_NAME'], os.getenv('EXECUTOR_NUMBER', 0))
 
-    if args.kubeadm:
+    if args.kubeadm is not None:
```

Comment:
Hunh, I thought I was having trouble with that syntax after changing this from a boolean, but it definitely works. I've reverted it back to the original "`if args.kubeadm:`".